### PR TITLE
add shreyalive

### LINKED ISSUE #297: Shreya Purohit
screenshot-github-com-shreyalive-1612212822111

### PR DESCRIPTION
**Pull request by shreyalive**
resolve #297

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -514,5 +514,11 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/48059117/106366095-a539be80-635f-11eb-95b0-c2d68c787410.png",
     "issueNumber": 285
-  }
+  },
+  {
+    "name": "Shreya Purohit",
+    "githubUsername": "shreyalive",
+    "imageUrl": "https://user-images.githubusercontent.com/63795399/106517095-07c8c100-64fe-11eb-807a-64405ddb4db0.png",
+    "issueNumber": 297
+}
```
Comment (AllanRegush) [inline data.json:523]: I think this bracket should be lined up like the rest :+1:

**Review by AllanRegush — COMMENTED** [review]
Looking Good :+1: Just a minor adjustment :nerd_face:

**Comment by shreyalive** [comment]
Thanks for the suggestions. Will correct it. 😄

**Review by shreyalive — COMMENTED** [review]
Placed the closing brackets in the right place. :)

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**